### ISSUE TITLE
A single word in the topic title may be broken at arbitrary points (default template css)

### ISSUE THREAD

**Issue by svver** [dotnet/docfx#234]
The default template CSS set word-break property to "break-all" for h1,h2 and so on, so the title may look ugly. For me [this](http://dotnet.github.io/docfx/tutorial/walkthrough_create_a_docfx_project.html) looks this way:
![image](https://cloud.githubusercontent.com/assets/8723144/14026940/167b5d02-f217-11e5-9c72-831cc5609214.png)
Is this a desired behaviour?

**Comment by simonmurdock** [comment]
I've noticed this too, it's definitely not desirable from my perspective. Maybe to accommodate very long unbroken strings in namespaces etc.

**Comment by sergey-vershinin** [comment]
@simonmurdock, indeed, [here](http://dotnet.github.io/docfx/api/Microsoft.CodeAnalysis.Workspaces.Dnx.html) is an example where a long namespace takes tho lines. 
![image](https://cloud.githubusercontent.com/assets/8723144/14043265/164ef00e-f2a4-11e5-8cd4-e6f7e0c27561.png)
If I set "word-break" to "normal" all the namespace goes to the new line. In case of a very long namespace this will be a problem. 

I would split namespaces using "." as a separator and then insert an `i` tag with a space and set font-size to 1px: 
```
Namespace Microsoft.<i> </i>CodeAnalysis.<i> </i>Workspaces.<i> </i>Dnx

<style>
   i {font-size: 1px}
</style>
```

**Comment by simonmurdock** [comment]
That makes sense. Would be good to move that "improve this doc" button somewhere else and reduce the font size a bit.

**Comment by hellosnow** [comment]
We just moved "improve this doc" to the right navigation via #626 , and we will have action items to refactor the word wrap.

**Comment by richardschneider** [comment]
How about surrounding the dot with U+200B ZERO WIDTH SPACE.  See #1222.

**Comment by hellosnow** [comment]
Sorry for the late updates, we fixed the title space by #1695 , the [github pages](http://dotnet.github.io/docfx/index.html) will be refreshed after v2.18.0 is released.

**Comment by hellosnow** [comment]
The code is in dev branch and not in master yet, the github pages will be refreshed after v2.19 is released.
